### PR TITLE
Proposed changes to revise current/ingest.html . . .

### PR DESCRIPTION
**Pull request by fbsolo**
Proposed changes for

https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html

to improve the text flow, style, and readability . . .

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by fbsolo** [comment]
I did sign the CLA a few minutes ago, but I am not a member of Elasticsearch. LMK what further steps I should take.

Thank you!

- Frank Solomon
  fbsolo@gmail.com

File: docs/reference/ingest.asciidoc [hunk 1]
```diff
@@ -3,26 +3,27 @@
...
 --------------------------------------------------
 node.ingest: false
 --------------------------------------------------
 
-To pre-process documents before indexing, you <<pipeline,define a pipeline>> that specifies
-a series of <<ingest-processors,processors>>. Each processor transforms the document in some way.
-For example, you may have a pipeline that consists of one processor that removes a field from
-the document followed by another processor that renames a field. Configured pipelines are then stored
-in the <<cluster-state,cluster state>>.
+To pre-process documents before indexing, define a pipeline that specifies a series of
```
Comment (cbuescher) [inline docs/reference/ingest.asciidoc]: Can you keep the links to the pipeline and ingest-processor documentation? I think they are useful.

File: docs/reference/ingest.asciidoc [hunk 2]
```diff
@@ -3,26 +3,27 @@
...
+processors. Each processor transforms the document in some specific way. For example, a
+pipeline might have one processor that removes a field from the document, followed by
+another processor that renames a field. The <<cluster-state,cluster state>> then stores
+the configured pipelines.
 
-To use a pipeline, you simply specify the `pipeline` parameter on an index or bulk request to
-tell the ingest node which pipeline to use. For example:
+To use a pipeline, simply specify the pipeline parameter on an index or bulk request. This
```
Comment (cbuescher) [inline docs/reference/ingest.asciidoc]: The backtics around "pipeline" shouldn't be removed here, we use them to highlight json or url parameters (like in this case).

**Review by cbuescher — CHANGES_REQUESTED** [review]
@fbsolo thanks a lot for your contribution, I think it really improves the current documentation. Your CLA signing looks good, however I left two small comments about things I'd like you to change, but once they are adressed I'm happy to merge it.

**Comment by fbsolo** [comment]
Done:

[https://github.com/elastic/elasticsearch/pull/28212/commits/0e333a7da8c8dedb9e825f142637b1db60b124a3]

**Comment by fbsolo** [comment]
[Done](https://github.com/elastic/elasticsearch/pull/28212/commits/0e333a7da8c8dedb9e825f142637b1db60b124a3).

**Review by cbuescher — APPROVED** [review]
@fbsolo thanks for the update, will merge this in.

**Pull request merged**